src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -2023,7 +2047,7 @@ private ImmutableArray<BoundExpression> BuildArgumentsForErrorRecovery(AnalyzedA
             var parameterListList = ArrayBuilder<ImmutableArray<ParameterSymbol>>.GetInstance();
             foreach (var m in methods)
             {
-                if (!IsUnboundGeneric(m) && m.ParameterCount > 0)
+                if (!IsUnboundGeneric(m) && m.GetParametersIncludingExtensionParameter().Length > 0)
                 {
                     parameterListList.Add(m.Parameters);
```

Comment:
>parameterListList.Add(m.Parameters); [](http://example.com/codeflow?start=20&length=36)

Shouldn't we also include extension parameter?

Reply:
Added and tried to observe the change, but I wasn't able to